src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -421,6 +425,7 @@ protected void closeResources() {
 
         if (results.isEmpty()) {
           consecutiveEmptyResults.compute(querier, (k, v) -> v + 1);
+          querier.resetRetryCount();
```

Comment:
Rather than reseting the retry count in both if and else clauses, does it make sense to reset the count before entering this if block?